focstest.py

```diff
@@ -103,6 +103,11 @@ def normalize_whitespace(text):
     """
     return ' '.join(text.split())
 
+def sorted_list(text):
+    """Normalize whitespace, then (crudely) parse output as a list and sort it"""
+    text = normalize_whitespace(text)
+    return text[text.index("[") + 1:text.rindex("]")].split(";").sort()
```

Comment:
I think this will return true for anything that doesn't match the brackets. I'd rather figure out the edge cases than ship something incomplete right now.